app/src/androidTest/java/com/duckduckgo/app/cta/ui/CtaViewModelTest.kt

```diff
@@ -490,34 +487,35 @@ class CtaViewModelTest {
     }
 
     @Test
-    fun whenRefreshCtaOnHomeTabAndCovidCtaNotShownThenCovidCtaTakesPrecedenceOverWidgetCta() = runBlockingTest {
+    fun whenRefreshCtaOnHomeTabAndCovidCtaNotShownThenWidgetCtaTakesPrecedenceOverCovidCta() = runBlockingTest {
         setNoFeatures()
+        whenever(mockWidgetCapabilities.supportsStandardWidgetAdd).thenReturn(true)
+        whenever(mockWidgetCapabilities.hasInstalledWidgets).thenReturn(false)
         whenever(mockDismissedCtaDao.exists(CtaId.ADD_WIDGET)).thenReturn(false)
         whenever(mockDismissedCtaDao.exists(CtaId.COVID)).thenReturn(false)
 
         val value = testee.refreshCta(coroutineRule.testDispatcher, isBrowserShowing = false)
-        assertTrue(value is HomeTopPanelCta.CovidCta)
+        assertTrue(value is HomePanelCta.AddWidgetInstructions)
     }
 
     @Test
-    fun whenRefreshCtaOnHomeTabAndConceptTestFeatureActiveAndIntroCtaWasPreviouslyShownThenCovidCtaShown() = runBlockingTest {
```

Comment:
I think we can remove this test or change it to test `DaxEndCta` is returned in that case, instead of false for `CovidCta`.

Reply:
Removed!